test/http.ts

```diff
@@ -7,6 +7,27 @@ import getStream = require('get-stream');
 import pEvent = require('p-event');
 import got, {HTTPError, UnsupportedProtocolError} from '../source';
 import withServer from './helpers/with-server';
+import os = require('os');
+
+const ifaces = os.networkInterfaces();
+let IPv6supported = false;
```

Comment:
```suggestion
const IPv6supported = Object.values(os.networkInterfaces()).some(entry => !entry.internal && entry.family === 'IPv6');
```

?

Reply:
That's not the same, an equivalent code would be:
```
const IPv6supported = Object.values(os.networkInterfaces()).some(iface => iface?.some(addr => !addr.internal && addr.family === 'IPv6'));
```